template/addons/next-auth/api-handler.ts

```diff
@@ -4,6 +4,13 @@ import CredentialsProvider from "next-auth/providers/credentials";
 import { env } from "../../../server/env";
 
 export const authOptions: NextAuthOptions = {
+  // Include user.id on session
+  callbacks: {
+    session({ session, user }) {
+      session.user.id = user.id;
```

Comment:
On second look. this won't do anything. there is no `user` when not using a database adapter. `token` may or may not have `id` in this case.

Reply:
Should I remove callbacks then for this case?

Reply:
@balazsorban44 if we only include this when Prisma is also included, would that make more sense?